src/placeos-frontend-loader/loader.cr

```diff
@@ -115,6 +115,11 @@ module PlaceOS::FrontendLoader
 
       case action
       in Action::Created, Action::Updated
+        # Skip load if the only change was `deployed_commit_hash`
+        if (changes = repository.changed_attributes).size == 1 && changes[:deployed_commit_hash]? && !repository.deployed_commit_hash.nil?
```

Comment:
```suggestion
        if (changes = repository.changed_attributes).size == 1 && changes[:deployed_commit_hash]?
```

it will already fail if there's a key present with the value of nil, no need for the extra check